sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Release History
 
 ## 1.0.0-beta.3 (Unreleased)
-
+- Add `getFormRecognizerClient()` and `getFormRecognizerAsyncClient()` in FormTrainingAsyncClient and FormTrainingClient
```

Comment:
Nit: The order feels off.

- Add `getFormRecognizerClient()` and `getFormRecognizerAsyncClient()` in FormTrainingClient and FormTrainingAsyncClient